tests/unit/ReportUtilsTest.js

```diff
@@ -182,16 +170,6 @@ describe('ReportUtils', () => {
             describe('Active', () => {
                 test('as member', () => {
                     expect(ReportUtils.getReportName({
```

Comment:
This looks like a mistake. We had two tests here:

1. One that asserted workspace members see the policy name
1. Another that asserted workspace admins see the member name

And now we have one (incorrectly named) test that asserts workspace admins see the member name

Reply:
Ah whoops I forgot to go back an put the member part of the test 🤦🏾  Yeah let me fix this.